### PR TITLE
Documentation for Presenter

### LINKED ISSUE #184: Document the files in mifopay/merchants package
**Summary:**

Document the files in mifopay/merchants package: https://github.com/openMF/mobile-wallet/tree/master/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants

### PR DESCRIPTION
**Pull request by shiv07tiwari**
Fixes #184

Please Add Summary of what changes you have made.(Optional)
Add documentation for merchants package
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by shiv07tiwari** [comment]
@ankurs287 @luckyman20 .
I tried fulfilling all the documentation guidelines in this PR.
Kindly tell me if I missed something.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/presenter/MerchantsPresenter.java [hunk 1]
```diff
@@ -60,6 +65,9 @@ public void onError(String message) {
                 });
     }
 
+    /**
+     * A method to fetch Merchants and Send them to UI Component.
+     */
```
Comment (ankurs287) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/presenter/MerchantsPresenter.java]: param not mentioned

**Comment by shiv07tiwari** [comment]
Made the changes @ankurs287 @luckyman20

**Comment by shiv07tiwari** [comment]
@ankur287 Do i need to add something else ?

**Comment by ankurs287** [comment]
@shiv07tiwari you could push directly to `master` for all the documentation related issues. Kindly do that.

**Comment by shiv07tiwari** [comment]
@ankurs287 So I have to make the PR for master instead of dev ?
Or i can directly commit in original repo ?

**Comment by ankurs287** [comment]
@shiv07tiwari Send a PR to `master` branch.

**Comment by shiv07tiwari** [comment]
Okay

**Pull request merged**